crypto/rand/rand_lcl.h

```diff
@@ -20,8 +20,17 @@
 /* How many times to read the TSC as a randomness source. */
 # define TSC_READ_COUNT                 4
 
-/* Maximum count allowed in reseeding */
-# define MAX_RESEED                     (1 << 24)
+/* Maximum reseed intervals */
+# define MAX_RESEED_INTERVAL                     (1 << 24)
+# define MAX_RESEED_TIME_INTERVAL                (1 << 20) /* approx. 12 days */
+
+/* Default reseed intervals */
+# define MASTER_RESEED_INTERVAL                  (1 << 8)
+# define SLAVE_RESEED_INTERVAL                   (1 << 16)
+# define MASTER_RESEED_TIME_INTERVAL             (60*60)   /* 1 hour */
+# define SLAVE_RESEED_TIME_INTERVAL              (60*60)   /* 1 hour */
```

Comment:
Should we go with the seven minute suggestion?

Reply:
Kurt, if you agree with the seven minute suggestion, then I will change the `SLAVE_RESEED_INTERVAL` to 7 minutes and squash all the fixup! commits after you finished reviewing.

Reply:
7 minutes would is fine for me.